src/training/flava_eval/finetune_glue.py

```diff
@@ -223,9 +269,8 @@ def train_one_epoch(model, data, epoch, optimizer, scheduler, early_stop, device
         label = batch["label"].to(device)
```

Comment:
Can you double check that this is correct for the tasks (e.g. no additional postprocessing to predictions or labels is necessary)?